CHANGES.md

```diff
@@ -6,9 +6,10 @@ Changes
 
 ### Unreleased
 
+* 2023-11-04 - Bugfix: Pass footer content without text_to_html div generation, solves #442.
+* 2023-11-03 - Bugfix: Add missing cachedef strings to the language pack, solves #441.
```

Comment:
I think this line should not be moved within your commit. Maybe it was moved during a rebase of yours?

Reply:
That was intentional.
We do sort chronologically backwards don't we?